spec/spec_helper.rb

```diff
@@ -2,7 +2,10 @@
 require 'simplecov-rcov'
 SimpleCov.formatter = SimpleCov::Formatter::RcovFormatter
 SimpleCov.minimum_coverage 100
-
+SimpleCov.start do
+  add_filter "/.bundle/"
```

Comment:
This is specific to individual configuration. I think `/gem/` would be more general.